src/content/general/organizations/index.md

```diff
@@ -0,0 +1,145 @@
+
+- Namespaces
+- Role based access control (RBAC)
+- Specific custom resources (CRs) that tie things together and make organizations easy to manage
+- Operators which automate some management tasks
+
+Typical use cases for organizations are:
+
+- Separating clusters for development, experimentation, and testing from clusters used for staging and for production purposes.
+
+- Isolating teams, business units, or even legal entities.
+
+At Giant Swarm, for example, we run several shared installations where we allow different customers access to a single organization only, usually before they get their own installation. This way we can ensure that each customer, while using the same management cluster, can only access their own clusters and other resources.
+
+### Visual overview {#intro-visual}
+
+[![Organizations in the Management API, visualized](organizations-management-api.svg)](organizations-management-api.svg)
+
+<!-- Source for above image: https://docs.google.com/drawings/d/1PDve3HoE7br_6npe0RSCw8ddt-H7pFztPlSDiQbNabs/edit -->
+
+## Current state and roadmap {#roadmap}
+
+Organizations are transitioning from being managed completely by microservices behind our REST API to becoming entities you can manage fully via the Management API. In this section we dissect where we are coming from, where we are heading, and what the current situation is on the different providers.
+
+- **2016 to present: REST API**
+
+    - Our REST API provides [functions to manage organizations](https://docs.giantswarm.io/api/#tag/organizations).
+    - Each workload cluster is owned by an organization.
+    - Customers assign users with proprietary Giant Swarm user accounts to organizations as members, in which they have full permissions.
+
+- **2019: introduction of the Management API**
+
+    - In order to give customers full access to management clusters, workload cluster and app resources, we provide experimental access to the Kubernetes API of management clusters, which we now call the Management API. (We used to call it the _Control Plane API_ back then.)
+    - For authentication we introduce OpenID Connect (OIDC), using the customer's own identity provider. We decide to abandon the proprietary Giant Swarm user account, used for the REST API, in the long run.
+    - In the beginning, the concept of the organization does not exist in the Management API.
+    - Cluster resources carry an annotation `giantswarm.io/organization` to indicate which organization they are assigned to via the REST API.
+
+- **2020: introduction of the organization concept in the Management API**
+
+    - With the [Organization CRD]({{< relref "/ui-api/management-api/crd/organizations.security.giantswarm.io.md" >}}) we introduce an entity in the Management API to represent an organization.
+    - Operators ensure that a namespace exists for each organization in the management cluster.
```

Comment:
and that there are RBAC roles defined that can be used to give users access to those organizations (or was that not rolled out in 2020 already?)